frontend/src/components/MakerForm/MakerForm.tsx

```diff
@@ -140,11 +140,17 @@ const MakerForm = ({
     updateCurrentPrice(limits.list, newCurrency, maker.premium);
 
     if (makerHasAmountRange) {
-      setMaker({
-        ...maker,
-        minAmount: parseFloat(Number(limits.list[newCurrency].max_amount * 0.25).toPrecision(2)),
-        maxAmount: parseFloat(Number(limits.list[newCurrency].max_amount * 0.75).toPrecision(2)),
-      });
+      const min_amount = parseFloat(Number(limits.list[newCurrency].min_amount).toPrecision(2));
```

Comment:
It seems to be the agreed JS style to use camelCase to name these consts, so probably best to rename them to `minAmount` and `maxAmount`